app/views/contact/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h1 Contact Us
+= form_for @contact, url: create_contact_url do |f|
+  = notice
+  = @contact.errors.full_messages.join(', ')
+  .row
+    .input-field.col.s12
+      = f.label :name
+      = f.text_field :name
+  .row
+    .input-field.col.s12
+      = f.label :email
+      = f.email_field :email
+  .row
+    .input-field.col.s12
```

Comment:
I'm not sure if the textarea styles are correct 🤔 

![image](https://user-images.githubusercontent.com/1099323/37983622-3391776a-31b1-11e8-85b2-ebe95619edb7.png)